source/main.cpp

```diff
@@ -129,7 +129,7 @@ int main(int argc, char **argv) {
 	int patchfile = 0;
 	int header;
 	rawDataOffset=0;
-	char datName[]="YS:/" DATNAME;
+	char datName[MAX_DATNAME_LEN]="YS:/" DATNAME;
```

Comment:
You may try clean the section first and strcpy the content to the buffer to do more.
I thought it was cause the wrong path you've installed get partially overwrited by the new one.
So there is a `bin` after the new path in the result path.